tutorial-content/content/example-step2.md

```diff
@@ -55,6 +56,29 @@ writeRaster(rr3, "../data/pcount/usap90ag.nc4",
  overwrite=TRUE, format="CDF", varname="Population", varunit="people",
  xname="lon", yname="lat")
 ```
+````
+ 
+````{tabbed} Python
+```{code-block} python
+#!pip install rasterio # will work in Jupyter Notebook
+import xarray as xr
+rr = xr.open_rasterio("../data/pcount/usap90ag.bil")
+
+#TODO rr2 <- aggregate(rr, fact=24, fun=sum)
```

Comment:
One solution here is to generate an appropriately sized numpy zeros matrix and then iterate through np.arange(ii, N, 24) in x and y, summing each into the matrix.